src/Nethermind/Nethermind.Synchronization/StateSync/PeerInfoExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Synchronization.Peers;
+
+namespace Nethermind.Synchronization.StateSync;
+
+public static class PeerInfoExtensions
+{
+    public static bool CanGetNodeData(this PeerInfo peerInfo)
+    {
+        return peerInfo.SyncPeer.ProtocolVersion < 67;
```

Comment:
can we use constant from `Eth67ProtocolHandler` for version? (There isn't one yet)

Reply:
Problem, Eth67ProtocolHandler is in `Nethermind.Network`, which depends on `Nethermind.Synchronization`, instead of the other way around, for some reason. I guess for `Protocol`, I could move them to `Nethermind.Blockchain`, but `Eth67ProtocolHandler` is a bit too specific.